File: 3010.将数组分成最小总代价的子数组-i.js
/*
 * @lc app=leetcode.cn id=3010 lang=javascript
 *
 * [3010] 将数组分成最小总代价的子数组 I
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumCost = function(nums) {
    nums = [nums[0], ...nums.slice(1).sort((a, b) => a - b)];
    return nums.slice(0, 3).reduce((sum, num) => sum + num, 0);
};
// @lc code=end

File: 3010.将数组分成最小总代价的子数组-i.java
/*
 * @lc app=leetcode.cn id=3010 lang=java
 *
 * [3010] 将数组分成最小总代价的子数组 I
 */

// @lc code=start
class Solution {
    public int minimumCost(int[] nums) {
        Arrays.sort(nums, 1, nums.length);
        return nums[0] + nums[1] + nums[2];
    }
}
// @lc code=end

